### PR TITLE
Upload illustrations from the May Book Dash

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Addresses an action #2301 

* Uploading images in reduced size, original images on Zenodo: https://zenodo.org/record/8169292

### List of changes proposed in this PR (pull-request)

* Uploaded images with text in smaller sizes

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by malvikasharan** [comment]
Alt text are being collected here: https://docs.google.com/spreadsheets/d/1B4mm-7ELWZCkHqE2rfeML1z803TNSnjLReMTlvyEltQ/edit#gid=0

File: book/website/figures/multi-tasking-community-manager.jpg
Comment (jcolomb) [inline book/website/figures/multi-tasking-community-manager.jpg:1]: this one has no text, should it be added anyway?

File: book/website/figures/five-year-partnership.jpg
Comment (jcolomb) [inline book/website/figures/five-year-partnership.jpg:1]: also a second version with TTW and Roche mentioned is below, should both be uploaded ?

**Review by jcolomb — CHANGES_REQUESTED** [review]
should we wait to have the alt text ?

2 exists in 2 versions, either comment or delete one version.

File: book/website/figures/multi-tasking-community-manager.jpg
Comment (malvikasharan) [reply book/website/figures/multi-tasking-community-manager.jpg:1]: ah probably not.

File: book/website/figures/five-year-partnership.jpg
Comment (malvikasharan) [reply book/website/figures/five-year-partnership.jpg:1]: Vicky will be using the one with logo in her case study, may be the one without text is not needed.

File: book/website/figures/multi-tasking-community-manager.jpg
Comment (malvikasharan) [reply book/website/figures/multi-tasking-community-manager.jpg:1]: Removed it.

**Comment by malvikasharan** [comment]
> should we wait to have the alt text ?

we can publish it, as the alt text will be used only in the chapters. We are also having a separate discussion on having a centralised location for all alt text, but this PR is not a blocker for that.

> 2 exists in 2 versions, either comment or delete one version.

I have removed the unlabeled community manager image. I will wait to hear from Vicky about the image she would like to keep for her chapter on academia-industry partnership.

Thank you for the prompt review Julien! 🥇

File: book/website/figures/five-year-partnership.jpg
Comment (vhellon) [reply book/website/figures/five-year-partnership.jpg:1]: Yes I will use the one with text please as it gives context for the research themes that are illustrated

**Review by da5nsy — COMMENTED** [review]
I'm assuming SVGs are not available?

If higher-quality bitmap versions exist that have not been jpeg compressed already, I'm happy to do my compression tricks (convert to png with indexed color)[1] to get better compression rates and higher image quality.

If not, LGTM!

[1]: _I really need to work out how to automate that..._ (throwback backlink: https://github.com/alan-turing-institute/the-turing-way/pull/2251#issuecomment-1219564091)

**Comment by jcolomb** [comment]
can we merge this now? waiting for the image for finishing #3111

File: book/website/figures/scientific-assessment.jpg
Comment (jcolomb) [inline book/website/figures/scientific-assessment.jpg:1]: can we rename this file: "why-open-hardware.jpg"

**Review by jcolomb — CHANGES_REQUESTED** [review]

**Review by jcolomb — APPROVED** [review]
all good now

**Pull request merged**